packages/shared/src/components/auth/AuthDefault.tsx

```diff
@@ -82,6 +83,8 @@ const AuthDefault = ({
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [shouldLogin]);
 
+  const { displayToast } = useToastNotification();
```

Comment:
Can you just move this to the top where all the other states are.
Should not be placed below render functions